### PR TITLE
BUG - HTML Checker Issues

### PR DESCRIPTION
**Pull request by frogpad7**
Homepage
- remove type from script
- remove the empty ID

Layouts
- remove role 'main' from main element

Getting Started
- fixing tags

Brand Colors
- fixing tags & id matches

Breadcrumbs
- removing copy attributes

Previous/Next Page Buttons
- changing ids

Scroll To Top Button
- changing ids

Add Data Button
- fixing element use

Footer
- fixing tags & removing extra stop element

Modal
- replacing h tag with p tag

Navbar
- removing id on gradient & replacing h tag with p tag & giving logos hidden text

Pagination
- removing empty attributes

Tabbed Content
- changing ids & removing partials

Switches
- simplifying attributes

Toggle Tokens
- providing better value properties

Address Form
- changing second city to Zip Code

User Id & Password Form
- escaping the left angle bracket ('<')

### CODE REVIEW

**Review by isralduke — DISMISSED** [review]
looks good

File: scss/_buttons.scss [hunk 1]
```diff
@@ -116,7 +116,7 @@
   background-color: transparent;
   border: 2px dashed $gray-600;
 
-  div {
+  .icon-holder {
```
Comment (james-alt) [inline scss/_buttons.scss:119]: I believe making this change is a breaking change. Most likely we need to leave the div here and then add .icon-holder as well.

**Review by james-alt — COMMENTED** [review]
We need to discuss this as it potentially introduces a breaking change for some.

File: docs/_includes/markup/form-template-address.njk [hunk 2]
```diff
@@ -40,9 +40,9 @@
       <div class="col-6">
         <div class="form-group">
           <label for="city">
-            <span class="fas fa-asterisk text-danger mr-1" aria-hidden="true"></span>City
+            <span class="fas fa-asterisk text-danger mr-1" aria-hidden="true"></span>Zip Code
```
Comment (james-alt) [inline docs/_includes/markup/form-template-address.njk:43]: The label for above is for `city` still and needs to be updated.

**Pull request closed without merging**